src/main/scala/org/broadinstitute/dsde/agora/server/business/AgoraBusiness.scala

```diff
@@ -128,6 +124,18 @@ class AgoraBusiness {
     }
   }
 
+  private def validateMethodRef(method: AgoraEntity, username: String) = {
+    if (!AgoraEntityPermissionsClient.getEntityPermission(method, username).canRead &&
+      !AgoraEntityPermissionsClient.getEntityPermission(method, "public").canRead) {
+      throw new AgoraEntityNotFoundException(method)
+    }
+    if (method.entityType.get == AgoraEntityType.Task) {
+      val methodIdentifier = "%s/%s/%s".format(method.namespace.get, method.name.get, method.snapshotId.get)
+      val message = "Configurations can only reference workflows, not tasks. Attempted to reference task %s".format(methodIdentifier)
```

Comment:
Can use scala string interpolation rather than .format:
s"Configurations can only reference workflows, not tasks. Attempted to reference task ${methodIdentifier}"